### ISSUE TITLE
What are these parameters and how they work?

### ISSUE THREAD

**Issue by adr-sit** [disdain13/PokeRoadie#222]
1. TransferTrimFatCount - Does it supersede "KeepDuplicateAmount" and / or the list in PokemonsNotToTransfer.ini?
2. PokeballRefillDelayMinutes
3. Min/Max Delays; does individual setting supersede MinDelay
   a. RecycleMin/MaxDelay
   b. PokedexEntryMin/MaxDelay
   c. LocationsMin/MaxDelay
   d. PokemonProcessDelayMinutes
4. TutorialCodename 

5 .TutorialGenerateCodename

**Comment by bulubebek** [comment]
some is already explained on wiki. tutorial code name = your in-game name. roadie able to do start and gym tutorial for you.

**Comment by pokeplayer2** [comment]
Look here: https://github.com/disdain13/PokeRoadie/wiki/Settings-&-Configs you can change the settings into settings.xml file.

**Comment by adr-sit** [comment]
Yes. I have gone through the Wiki and I still do not quite understand what they mean and how they work. Thanks anyone who can explain them in some details.

I think I know what PokedexEntry is doing and I won't change it. But what can I achieve by increasing delay value of 
1. PokeballRefill 
2. Recycle
3. Locations
4. PokemonProcess

Also, how does "TransferTrimFatCount" work with  KeepDuplicateAmount" and PokemonsNotToTransfer.ini?

Thanks.
